homeassistant/components/sensor/nut.py

```diff
@@ -209,11 +215,12 @@ def unit_of_measurement(self):
     def device_state_attributes(self):
         """Return the sensor attributes."""
         attr = dict()
-        attr[ATTR_STATE] = self.opp_state()
+        attr[ATTR_STATE] = "{} ({})".format(self.display_state(),
```

Comment:
If you are very accurate this is a breaking change. I would remove the duplicate info here. What do you think?

Reply:
Yes, you are right. And with the introduction of a separate sensor type for the human-readable status, the additional information in the more info dialogue is probably not necessary anymore.